ops/terraform/services/eft/lambda_src/sftp_outbound_transfer/dev-requirements.txt

```diff
@@ -2,85 +2,85 @@
 # This file is autogenerated by pip-compile with Python 3.12
 # by the following command:
 #
-#    pip-compile --allow-unsafe --constraint=requirements.txt --generate-hashes --output-file=dev-requirements.txt dev-requirements.in
+#    pip-compile --allow-unsafe --generate-hashes --output-file=dev-requirements.txt dev-requirements.in
```

Comment:
Apologies for the late addition–I just noticed the modest difference here and later in _this_ requirements file as a result. Could you re-institute the constraints flag and re-generate?
```suggestion
#    pip-compile --allow-unsafe --constraint=requirements.txt --generate-hashes --output-file=dev-requirements.txt dev-requirements.in
```

Reply:
complete